go.sum

```diff
@@ -3,26 +3,38 @@ cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
```

Comment:
Similar note as the other go.sum file - try running `go mod tidy`

Reply:
it was done before committing the code